### PR TITLE
[SPARK-344] Revert "Implement a Go Spark CLI"

### PR DESCRIPTION
**Pull request by nickbp**
This reverts commit 7196a62637ee116d580334bbf18d364a0726d1a6.

### CODE REVIEW

**Comment by mgummelt** [comment]
@nickbp CI is fixed, so we should be able to merge this soon.  please fix conflict first.

**Comment by mgummelt** [comment]
retest this please

**Comment by nickbp** [comment]
retest this please

**Comment by nickbp** [comment]
green! rebasing.

File: bin/jenkins-package-test.sh [hunk 1]
```diff
@@ -14,6 +14,12 @@ function run() {
 }
 
 pushd "${SPARK_BUILD_DIR}"
+
+export VERSION=${ghprbActualCommit}
```
Comment (mgummelt) [inline bin/jenkins-package-test.sh]: Remove this stanza.  Not needed.

File: bin/universe.sh [hunk 2]
```diff
@@ -24,6 +25,13 @@ configure_env() {
         echo "Using Spark dist URI: $SPARK_DIST_URI"
     fi
 
+    if [ -z "${CLI_VERSION}" ]; then
```
Comment (mgummelt) [inline bin/universe.sh]: Aren't we versioning the CLIs the same as the frameworks?

File: bin/universe.sh [hunk 3]
```diff
@@ -72,19 +80,17 @@ upload_cli_and_stub_universe() {
...
             ${BASEDIR}/package/ \
-            ${BASEDIR}/cli/dcos-spark/dcos-spark-darwin \
-            ${BASEDIR}/cli/dcos-spark/dcos-spark-linux \
-            ${BASEDIR}/cli/dcos-spark/dcos-spark.exe \
-            ${BASEDIR}/cli/python/dist/*.whl
+            ${BASEDIR}/cli/dist/*.whl
 }
 
-# set SPARK_URI and DOCKER_IMAGE:
+# set CLI_VERSION, SPARK_URI, and DOCKER_IMAGE:
```
Comment (mgummelt) [inline bin/universe.sh]: move this doc above `configure_env`.  (that's what it documents)

File: bin/universe.sh (hunk 2 above)
Comment (nickbp) [inline bin/universe.sh]: The CLI version was a separate field in the manifest.

Comment (mgummelt) [inline bin/universe.sh]: Let's remove the option to customize it via an env var.  That shouldn't be needed.

File: bin/universe.sh (hunk 3 above)
Comment (nickbp) [inline bin/universe.sh]: Done

File: bin/universe.sh (hunk 2 above)
Comment (nickbp) [inline bin/universe.sh]: Done

**Pull request merged**